### DISCUSSION TITLE
How to Improve Github Profile?

### DISCUSSION THREAD

**Discussion by AbuBakkarRaihan** [community/community#136769] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey can anyone tell me how I can create a good GitHub profile? Actually on GitHub I don't get any forks, stars, pull requests, or anything. If you have a good opinion you can let me know!

**Comment by R-I-T-U** [comment, marked as answer]
Your profile already looks amazing, but there are only two repositories available. I think you should focus on building more projects.....? Just a thought.

**Comment by Ammar-Alnagar** [comment]
Your profile page looks great,  you need to create more repos / fork .

**Reply by AbuBakkarRaihan** [reply]
Bro Thanks For Your Opinions 😍

**Comment by ebndev** [comment]
Hi @AbuBakkarRaihan, thanks for being a part of the GitHub Community!

You can check out our [Showcase Your GitHub Profile! 🌟](https://github.com/orgs/community/discussions/121833) thread for more inspiration! There are plenty of great profiles there. Feel free to post yours when you're ready as well 😄
